### ISSUE TITLE
[4.x] atum contrast [a11y]

### ISSUE THREAD

**Issue by brianteeman** [joomla/joomla-cms#37919]
When the class text-muted is used for text it fails the contrast checks.

At the moment I can only see this scenario occuring with the new MFA but it is probably occuring elswehere as well.

![image](https://user-images.githubusercontent.com/1296369/170857518-036c19fc-972b-473d-b0f0-e5a6afc3e2ec.png)

Element has insufficient color contrast of 3.74 (foreground color: #808589, background color: #ffffff, font size: 9.6pt (12.8px), font weight: normal). Expected contrast ratio of 4.5:1

possible solution

Use foreground color: #72777b and the original background color: #ffffff to meet a contrast ratio of 4.53:1.

[WCAG 1.4.3](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html)

**Comment by brianteeman** [comment]
Same page same contrast issue

![image](https://user-images.githubusercontent.com/1296369/170857779-2f2854c8-3c02-4c4f-80b0-61953371273c.png)

**Comment by chmst** [comment]
Disabled Action Buttons also have bad contrast.
